src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -334,62 +332,44 @@ const ImageSelector = ({ control, field, item, selectImage }) => {
         control={control}
         errorMessage={errorMessage}
         hidden
+        validate
         name={name}
-        validate={true}
         value={value}
       />
       <RegularText smallest placeholder>
         {infoText}
       </RegularText>
 
       {value ? (
-        <WrapperRow center spaceBetween>
-          <Image source={{ uri: value }} style={styles.image} />
-
-          <TouchableOpacity
-            onPress={() => {
-              onChange('');
-              setErrorMessage('');
-            }}
-          >
-            <Icon.Trash color={colors.error} size={normalize(16)} />
-          </TouchableOpacity>
-        </WrapperRow>
+        <>
+          <WrapperRow center spaceBetween>
+            <Image source={{ uri: value }} style={styles.image} />
+
+            <TouchableOpacity
+              onPress={() => {
+                onChange('');
+                setErrorMessage('');
```

Comment:
so we also need to reset the `setImageInfoText` to `''` here?

Reply:
After the picture was deleted, I did not delete it because it was not displayed on the screen. But it can be deleted.